packages/tests/server/react/createClient.test.tsx

```diff
@@ -67,57 +66,3 @@ test('createClient()', async () => {
     expect(utils.container).toHaveTextContent('world');
   });
 });
-
-test('useDehydratedState() - internal', async () => {
```

Comment:
👀

Reply:
killed it, it's only used for `withTRPC`